packages/ui/src/styles.css

```diff
@@ -422,7 +422,7 @@
   }
 }
 
-:focus-visible {
+:where(:focus-visible) {
```

Comment:
Why was this added?

Reply:
As we are adding `:where` in [button plugin](https://github.com/harness/canary/pull/1494/files#diff-1e2a8823f651352087e33971c754c061f2b0b1a67fad50e0888988c4d1a24a3cR159), global `:focus-visible` was overriding it if it is without `:where`. Adding `:where` to remove its specificity.